### ISSUE TITLE
could not find default credentials

### ISSUE THREAD

**Issue by skorphil** [daichirata/gcsproxy#23]
How to specify path to application_default_credentials.json ? I just downloaded em to my PC, but still got error when launching docker image with gcsproxy

**Comment by skorphil** [comment]
Well i mounted em  PC:~/.config/gcloud -> Container:/root/.config/gcloud

But seems like proxy not working. i got an error 
```
This site can’t be reached127.0.0.1 refused to connect.
Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
ERR_CONNECTION_REFUSED 
```
while trying to open file http://127.0.0.1:8080/bucket/pathtofile

**Comment by daichirata** [comment]
Have you tried setting up Application Default Credentials?

https://cloud.google.com/docs/authentication/provide-credentials-adc

**Comment by skorphil** [comment]
> Have you tried setting up Application Default Credentials?
> 
> https://cloud.google.com/docs/authentication/provide-credentials-adc

Yes, I generated .json with a command `gcloud auth application-default login`
Then I mount `~/.config/gcloud` to `/root/.config/gcloud` in docker container.
Docker container has launched, but I got error in browser `ERR_CONNECTION_REFUSED` either I connect to `localhost:8080` or any existing path to file i.e. `http://127.0.0.1:8080/bucket/file`

IDK how can I diagnose the problem. `var/logs/` in the container does not contain any error logs
